tests/python_client/README_CN.md

```diff
@@ -65,7 +65,7 @@ Milvus 支持4种部署方式，请根据需求选择部署方式，PyMilvus 支
 
 ### PyMilvus 测试环境部署及用例执行
 
-推荐使用 **Python 3(>= 3.8)** ，与 **[PyMilvus](https://pymilvus.readthedocs.io/en/latest/install.html)** 支持的 Python 版本保持一致。
+推荐使用 **Python 3(>= 3.8 or <= 3.11)** ，与 **[PyMilvus](https://pymilvus.readthedocs.io/en/latest/install.html)** 支持的 Python 版本保持一致。
```

Comment:
Please remove this change , `tests/python_client` doesn't need conan so don't need to limit the upper version.

Reply:
ok, done